Makefile

```diff
@@ -0,0 +1,5 @@
+requirements:
+	@echo "Requirements are installed through edx-platform"
+
+test:
+	@echo "Tests are contained in edx-platform. See openedx/tests/xblock_integration/test_review_xblock.py for tests"
```

Comment:
We've discussed this - but I'm not a fan of completely punting on all tests in this repo. Certain integration-like tests might only be runnable within edx-platform (without a ton of effort). But it's possible to unit-test code via mocking and achieve some level of code coverage in this repo.

For an example, look at the edx-ora2 XBlock tests - here's a pointer to just some of the testing in that repo:
https://github.com/edx/edx-ora2/tree/master/openassessment/xblock/test

Reply:
I'm happy to talk about this more, but after discussing it with Eric, it seems like this repos version of unit tests are essentially integration tests because of the dependencies it needs. In order to mock what I need, it would require 2 courses, a number of problems, and a database. At the point I have all of that in my repo, I'm recreating enough of platform that I think it would just be easier to test it there.